src/TopApp.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import axios from 'axios';
+import { Container } from 'semantic-ui-react';
+import { ToastContainer } from 'react-toastify';
+import App from './App';
+// import PrivateRoute from './PrivateRoute';
+import { UserContext, userContext } from './userContext';
+import 'react-toastify/dist/ReactToastify.css';
+
+axios.defaults.baseURL = 'http://localhost:4000/api';
+
+export default function TopApp() {
+  // Example of setting isAuthenticated in userContext, This function is for demonstration.
```

Comment:
We can use github CI to remove prettier/eslint errors. Why we need these comments?

Reply:
Could you please provide some examples? Not aware about that.

Reply:
We can use github actions to validate some errors before merging feature branch into master

[example ci configuration](https://github.com/sveltejs/svelte/blob/master/.github/workflows/ci.yml)